[2024-09-27T18:10:32.360+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-27T18:10:32.510+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-09-27T18:10:32.566+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-09-27T18:10:32.584+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-27T18:10:32.672+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): create_share_prices_table> on 2023-10-01 00:00:00+00:00
[2024-09-27T18:10:32.700+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=11880) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-27T18:10:32.744+0000] {standard_task_runner.py:72} INFO - Started process 11922 to run task
[2024-09-27T18:10:32.744+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_share_prices_with_company_data', 'create_share_prices_table', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '1129', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpfwi4pevi']
[2024-09-27T18:10:32.774+0000] {standard_task_runner.py:105} INFO - Job 1129: Subtask create_share_prices_table
[2024-09-27T18:10:33.090+0000] {task_command.py:467} INFO - Running <TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [running]> on host 13b5797a93ab
[2024-09-27T18:10:33.787+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrape_and_save_share_prices_with_company_data' AIRFLOW_CTX_TASK_ID='create_share_prices_table' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2024-09-27T18:10:33.791+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-27T18:10:33.804+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-27T18:10:33.809+0000] {logging_mixin.py:190} INFO - Current task name:create_share_prices_table state:running start_date:2024-09-27 18:10:32.514676+00:00
[2024-09-27T18:10:33.810+0000] {logging_mixin.py:190} INFO - Dag name:scrape_and_save_share_prices_with_company_data and current dag run status:running
[2024-09-27T18:10:33.811+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-27T18:10:33.911+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T18:10:33.978+0000] {sql.py:509} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS company_share_prices (
            id SERIAL,
            company_id INTEGER NOT NULL,
            date DATE,
            latest_trading_price VARCHAR(50),
            low_price VARCHAR(50),
            high_price VARCHAR(50),
            closing_price VARCHAR(50),
            ycp VARCHAR(50),
            change VARCHAR(10),
            trade VARCHAR(50),
            value VARCHAR(50),
            volume VARCHAR(50),
            PRIMARY KEY(company_id, id),
            FOREIGN KEY(company_id)
                REFERENCES company_list(company_id)
                ON UPDATE CASCADE ON DELETE CASCADE
        );
    , parameters: None
[2024-09-27T18:10:33.989+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-27T18:10:34.045+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-27T18:10:34.064+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=scrape_and_save_share_prices_with_company_data, task_id=create_share_prices_table, run_id=scheduled__2023-10-01T00:00:00+00:00, execution_date=20231001T000000, start_date=20240927T181032, end_date=20240927T181034
[2024-09-27T18:10:34.131+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-27T18:10:34.137+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-27T18:10:34.138+0000] {logging_mixin.py:190} INFO - Dag name:scrape_and_save_share_prices_with_company_data queued_at:2024-09-27 18:06:50.941856+00:00
[2024-09-27T18:10:34.139+0000] {logging_mixin.py:190} INFO - Task hostname:13b5797a93ab operator:PythonOperator
[2024-09-27T18:10:34.210+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-27T18:10:34.325+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-09-27T21:52:20.695+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-27T21:52:20.741+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-09-27T21:52:20.787+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-09-27T21:52:20.788+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-27T21:52:20.941+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): create_share_prices_table> on 2023-10-01 00:00:00+00:00
[2024-09-27T21:52:20.970+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_share_prices_with_company_data', 'create_share_prices_table', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '2152', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpwp_3ui28']
[2024-09-27T21:52:20.973+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=489) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-27T21:52:20.984+0000] {standard_task_runner.py:105} INFO - Job 2152: Subtask create_share_prices_table
[2024-09-27T21:52:20.984+0000] {standard_task_runner.py:72} INFO - Started process 505 to run task
[2024-09-27T21:52:21.170+0000] {task_command.py:467} INFO - Running <TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [running]> on host fd62fe12432d
[2024-09-27T21:52:21.544+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrape_and_save_share_prices_with_company_data' AIRFLOW_CTX_TASK_ID='create_share_prices_table' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2024-09-27T21:52:21.547+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-27T21:52:21.549+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-27T21:52:21.550+0000] {logging_mixin.py:190} INFO - Current task name:create_share_prices_table state:running start_date:2024-09-27 21:52:20.742726+00:00
[2024-09-27T21:52:21.551+0000] {logging_mixin.py:190} INFO - Dag name:scrape_and_save_share_prices_with_company_data and current dag run status:running
[2024-09-27T21:52:21.553+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-27T21:52:21.588+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T21:52:42.654+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dag.py", line 108, in create_share_prices_table
    hook.run(create_table_query)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 452, in run
    with self._create_autocommit_connection(autocommit) as conn:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 574, in _create_autocommit_connection
    with closing(self.get_conn()) as conn:
                 ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/hooks/postgres.py", line 176, in get_conn
    self.conn = psycopg2.connect(**conn_args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?

[2024-09-27T21:52:42.726+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-09-27T21:52:42.728+0000] {logging_mixin.py:190} INFO - Task start:2024-09-27 21:52:20.742726+00:00 end:2024-09-27 21:52:42.725622+00:00 duration:21.982896
[2024-09-27T21:52:42.729+0000] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): create_share_prices_table> dag:<DAG: scrape_and_save_share_prices_with_company_data> dagrun:<DagRun scrape_and_save_share_prices_with_company_data @ 2023-10-01 00:00:00+00:00: scheduled__2023-10-01T00:00:00+00:00, state:running, queued_at: 2024-09-27 21:51:00.397983+00:00. externally triggered: False>
[2024-09-27T21:52:42.730+0000] {logging_mixin.py:190} INFO - Failure caused by connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T21:52:42.734+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=scrape_and_save_share_prices_with_company_data, task_id=create_share_prices_table, run_id=scheduled__2023-10-01T00:00:00+00:00, execution_date=20231001T000000, start_date=20240927T215220, end_date=20240927T215242
[2024-09-27T21:52:42.938+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-27T21:52:42.953+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 2152 for task create_share_prices_table (connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
; 505)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dag.py", line 108, in create_share_prices_table
    hook.run(create_table_query)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 452, in run
    with self._create_autocommit_connection(autocommit) as conn:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/contextlib.py", line 137, in __enter__
    return next(self.gen)
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py", line 574, in _create_autocommit_connection
    with closing(self.get_conn()) as conn:
                 ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/hooks/postgres.py", line 176, in get_conn
    self.conn = psycopg2.connect(**conn_args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?

[2024-09-27T21:52:43.003+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-09-27T21:52:43.042+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-27T21:52:43.202+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-09-27T22:42:49.871+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-27T22:42:49.907+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-09-27T22:42:49.928+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2024-09-27T22:42:49.929+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-27T22:42:49.970+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): create_share_prices_table> on 2023-10-01 00:00:00+00:00
[2024-09-27T22:42:49.986+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=714) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-27T22:42:49.989+0000] {standard_task_runner.py:72} INFO - Started process 723 to run task
[2024-09-27T22:42:49.991+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_share_prices_with_company_data', 'create_share_prices_table', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '2713', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpi1hvrk94']
[2024-09-27T22:42:49.993+0000] {standard_task_runner.py:105} INFO - Job 2713: Subtask create_share_prices_table
[2024-09-27T22:42:50.127+0000] {task_command.py:467} INFO - Running <TaskInstance: scrape_and_save_share_prices_with_company_data.create_share_prices_table scheduled__2023-10-01T00:00:00+00:00 [running]> on host 7746de5121f6
[2024-09-27T22:42:50.386+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrape_and_save_share_prices_with_company_data' AIRFLOW_CTX_TASK_ID='create_share_prices_table' AIRFLOW_CTX_EXECUTION_DATE='2023-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-01T00:00:00+00:00'
[2024-09-27T22:42:50.399+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-27T22:42:50.400+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-27T22:42:50.405+0000] {logging_mixin.py:190} INFO - Current task name:create_share_prices_table state:running start_date:2024-09-27 22:42:49.908449+00:00
[2024-09-27T22:42:50.409+0000] {logging_mixin.py:190} INFO - Dag name:scrape_and_save_share_prices_with_company_data and current dag run status:running
[2024-09-27T22:42:50.412+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-27T22:42:50.449+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T22:42:50.466+0000] {sql.py:509} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS company_share_prices (
            id SERIAL,
            company_id INTEGER NOT NULL,
            date DATE,
            latest_trading_price VARCHAR(50),
            low_price VARCHAR(50),
            high_price VARCHAR(50),
            closing_price VARCHAR(50),
            ycp VARCHAR(50),
            change VARCHAR(10),
            trade VARCHAR(50),
            value VARCHAR(50),
            volume VARCHAR(50),
            PRIMARY KEY(company_id, id),
            FOREIGN KEY(company_id)
                REFERENCES company_list(company_id)
                ON UPDATE CASCADE ON DELETE CASCADE
        );
    , parameters: None
[2024-09-27T22:42:50.469+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-27T22:42:50.523+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-27T22:42:50.533+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=scrape_and_save_share_prices_with_company_data, task_id=create_share_prices_table, run_id=scheduled__2023-10-01T00:00:00+00:00, execution_date=20231001T000000, start_date=20240927T224249, end_date=20240927T224250
[2024-09-27T22:42:50.573+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-27T22:42:50.576+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-27T22:42:50.580+0000] {logging_mixin.py:190} INFO - Dag name:scrape_and_save_share_prices_with_company_data queued_at:2024-09-27 22:34:36.975215+00:00
[2024-09-27T22:42:50.581+0000] {logging_mixin.py:190} INFO - Task hostname:7746de5121f6 operator:PythonOperator
[2024-09-27T22:42:50.630+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-27T22:42:50.669+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
